src/utils/helpers/date/date.js

```diff
@@ -89,8 +106,8 @@ const DateHelper = {
    * @param {String} locale - defaulted to I18n.locale
    * @return {Array}
    */
-  weekdaysMinified: () => {
```

Comment:
Please document this as a breaking change.

Reply:
added, thanks